docs/overview/header.md

```diff
@@ -5,3 +5,74 @@ to the top of the viewport even when the main content area is scrolled up and do
 In order to create a header you have to create a CDI bean that implement org.uberfire.client.workbench.Header interface.
 
 ![header](header.png)
+
+Let us add a simple Composite that adds 3 labels. 
+
+In package org.uberfire.client.navbar we can create 
+
+- HeaderView.java
+
+```
+package org.uberfire.client.navbar;
+
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HorizontalPanel;
+import com.google.gwt.user.client.ui.Label;
+
+public class HeaderView extends Composite {
+
+    public HeaderView() {
```

Comment:
nheron, I think the best way to do it is to write a sample for you in your repo and then explain it. Could you please share commit rights for me in your fork? Or feel free to ping me on irc.